{"name": "test_get_token", "status": "broken", "statusDetails": {"message": "FileNotFoundError: [Errno 2] No such file or directory: '../api/wework.yaml'", "trace": "self = <WechatAPI.testcase.test_work.TestWework object at 0x000001F115721080>\n\n    def test_get_token(self):\n>       print(Wework().test_get_member(\"138550817\"))\n\nWechatAPI\\testcase\\test_work.py:16: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nself = <WechatAPI.api.we_work.Wework object at 0x000001F115721278>\n\n    def __init__(self):\n        self.token = Util().token() #不要忘了Util的()\n        self.params[\"token\"]=self.token #要记得替换token的变量\n    \n>       with open(\"../api/wework.yaml\",encoding=\"utf-8\")as f:\nE       FileNotFoundError: [Errno 2] No such file or directory: '../api/wework.yaml'\n\nWechatAPI\\api\\we_work.py:27: FileNotFoundError"}, "start": 1647486667144, "stop": 1647486667847, "uuid": "9c5b319e-4df0-41c1-b9bb-9d39ae18e486", "historyId": "620ab8e5e6f59bb65d84e9f07968fda6", "testCaseId": "deaefd99f50a42b3be6ec70947f6178b", "fullName": "WechatAPI.testcase.test_work.TestWework#test_get_token", "labels": [{"name": "parentSuite", "value": "WechatAPI.testcase"}, {"name": "suite", "value": "test_work"}, {"name": "subSuite", "value": "TestWework"}, {"name": "host", "value": "ZXJ"}, {"name": "thread", "value": "58484-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "WechatAPI.testcase.test_work"}]}